index.html

```diff
@@ -5,9 +5,12 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width">
 
-    <link href="//fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
+    <link href="http://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
     <link rel="stylesheet" href="styles/normalize.css">
     <link rel="stylesheet" href="styles/main.css">
+    <link rel="icon" type="image/png" href="images/favicon.png">
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
```

Comment:
I was expecting this comment.

I am aware that JQuery is better to avoid. The main reason I used JQuery was that it wasn't giving me that CORs error I describe in a later comment. But also, before this my Js knowledge was terrible. (Still pretty much is) I just wanted to get something working.

I'd be happy to try remove JQuery if you can tell me how to make a successful CORs request.